bot/decorators.py

```diff
@@ -3,15 +3,21 @@
 from discord.ext.commands import Context
 
 
+def invoke_with_prefix(prefix: str) -> commands.check:
```

Comment:
this doesnt fit all use cases. how about `ignore_prefix` and `allow_prefix` , and they both take lists.that gives us enough configurability to suit all cases without being terrible for readability.

Reply:
by default, it'll ignore all other prefixes, I'll change it so it'll take a sequence of arguments considered prefixes and just apply a presence check